homeassistant/components/light/mqtt/schema_template.py

```diff
@@ -58,20 +58,17 @@
     vol.Optional(CONF_STATE_TEMPLATE): cv.template,
     vol.Optional(CONF_STATE_TOPIC): mqtt.valid_subscribe_topic,
     vol.Optional(CONF_WHITE_VALUE_TEMPLATE): cv.template,
-    vol.Required(CONF_COMMAND_OFF_TEMPLATE): cv.template,
-    vol.Required(CONF_COMMAND_ON_TEMPLATE): cv.template,
+    vol.Optional(CONF_COMMAND_OFF_TEMPLATE): cv.template,
+    vol.Optional(CONF_COMMAND_ON_TEMPLATE): cv.template,
```

Comment:
To validate the schema, can a custom validator be used to make sure CONF_COMMAND_ON_TEMPLATE and CONF_COMMAND_OFF_TEMPLATE are both thereif 'schema' == 'template', or what is the best way?